### ISSUE TITLE
Project Organizer: Remember "Search in full path" setting

### ISSUE THREAD

**Issue by chrisgraham** [geany/geany-plugins#908]
Project Organizer has a "Search in full path" checkbox in the "Find Project File" dialog.

The value of the checkbox is lost when Geany is re-opened, it will go back to unchecked.

Ideally the value should be remembered, as many (but not all) developers will have a preference for it being checked.

**Comment by techee** [comment]
Yeah, there is currently no settings of the plugin itself (all the settings are saved on a project basis) and yeah, I've been thinking about it myself too. I just didn't find it annoying enough to force me to implement it ;-).
